src/views/Room.vue

```diff
@@ -93,12 +93,14 @@ export default {
 
       rtc.on("signaling_not_reachable", () => {
         logger.error("signaling server not reachable")
-        this.reconnectRtc()
+        // this.reconnectRtcWhenOnLine()
       })
 
-      rtc.on("signaling_error", (error) => {
-        logger.error("signaling error", error)
-        this.reconnectRtc()
+      rtc.on("signaling_error", (errorType, error) => {
+        logger.error("signaling error", errorType, error)
+        if(errorType === "socket" || errorType === "missing_pongs") {
```

Comment:
The other two so far are "server" and "invalid_json". Since server events can also be of very different nature, we could also add an error type field to server error events (e.g. the left room event should not trigger reconnects, but others might)